src/components/Feedback/FeedbackThanks.tsx

```diff
@@ -32,7 +33,7 @@ function FeedbackThanks({ classes }: Props) {
       <Typography className={classes.text}>
         If you have further questions or feedback please feel free to{' '}
         <a
-          href="mailto:dtpr-guide@helpfulplaces.com"
+          href={`mailto:dtpr-guide@helpfulplaces.com?subject=Further Feedback: ${sensor.name} - ${place.name}&body=%0d%0a%0d%0a%0d%0a%0d%0a%0d%0aWe use your information to answer your questions and to aggregate feedback on the use of DTPR and the technology that the open-source communication is used to describe.  For more information about the DTPR standard, visit https://dtpr.helpfulplaces.com/.%0d%0a%0d%0aIt helps us to know what technology you are referring to, please leave this message at the bottom of your email:%0d%0aThe feedback being provided is for the technology on this page: ${window.location.href}`}
```

Comment:
Have you played around with the formatting here at all? I tried the link and normally it would automatically pick up the links within the body but it's not in this case. Could be my email client though.